doc/HttpLuaModule.wiki

```diff
@@ -5581,6 +5581,73 @@ According to the current implementation, only the parent coroutine (or "light th
 
 This API was first enabled in the <code>v0.9.9</code> release.
 
+== ngx.semaphore.new ==
+'''syntax:''' ''sem, err = ngx.semaphore.new(n)''
+
+'''context:''' ''init_worker_by_lua*, set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, body_filter_by_lua, log_by_lua*, ngx.timer.*''
+
+Create a semaphore that has n resource. LuaJIT's ffi is need by this api.
```

Comment:
Grammar: "Create" -> "Creates". We need `<code>` around `n`. Also, FFI is preferred over ffi.